lib/CMakeLists.txt

```diff
@@ -121,6 +121,7 @@ set(LLVM_INCLUDE_BENCHMARKS OFF)
 set(LLVM_INCLUDE_TESTS OFF)
 set(LLVM_TOOL_REMARKS_SHLIB_BUILD OFF)
 set(LLVM_ENABLE_ZLIB OFF)
+set(LVM_ENABLE_ZLIB:STRING OFF)
```

Comment:
Why does it say `LVM` instead of `LLVM`?

Reply:
oops. and yet it still worked. will fix on main.

Reply:
i probably fat fingered on creating the PR. we need a test for this eventually, but it requires creating a program that gets linked and making sure it builds.